Cargo.lock

```diff
@@ -2331,18 +2337,28 @@ dependencies = [
 
 [[package]]
 name = "object"
-version = "0.30.1"
+version = "0.30.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "8d864c91689fdc196779b98dba0aceac6118594c2df6ee5d943eb6a8df4d107a"
+checksum = "ea86265d3d3dcb6a27fc51bd29a4bf387fae9d2986b823079d4986af253eb439"
 dependencies = [
  "compiler_builtins",
+ "memchr",
+ "rustc-std-workspace-alloc",
+ "rustc-std-workspace-core",
+]
```

Comment:
There also a minor bump of std one's object dependency.

Reply:
Well, this doesn't matter because backtrace and std still agree.

Reply:
I downgraded object to 0.30.1 anyways just to minimize later merge conflicts.